user exception caught
error call caught
no method error
test/pkg/base/exceptionsrun001.hs:38:1-13: Non-exhaustive patterns in function test1
test/pkg/base/exceptionsrun001.hs:45:1-26: Non-exhaustive patterns in function test2

